ros2_ws/src/warehouse_robot/launch/launch_sim.launch.py

```diff
@@ -58,18 +58,18 @@ def generate_launch_description():
     diff_drive_spawner = Node(
         package="controller_manager",
         executable="spawner",
-        arguments=["diff_cont", "--controller-manager-timeout", "30"],
+        arguments=["diff_cont"],
     )
 
     joint_broad_spawner = Node(
         package="controller_manager",
         executable="spawner",
-        arguments=["joint_broad", "--controller-manager-timeout", "30"],
+        arguments=["joint_broad"],
```

Comment:
Same with leaving this timeout in
```suggestion
        arguments=["joint_broad", "--controller-manager-timeout", "30"],
```